indico/MaKaC/webinterface/pages/category.py

```diff
@@ -180,62 +197,39 @@ def __init__( self, aw, entry ):
         self._break = entry
         self._aw = aw
 
-    def getVars( self ):
-        tz = DisplayTZ(self._aw).getDisplayTZ()
-        vars = wcomponents.WTemplated.getVars( self )
-        vars["startTime"] = self._break.getAdjustedStartDate(tz).strftime("%H:%M")
-        vars["title"] = self._break.getTitle()
+    def getVars(self):
+        vars = wcomponents.WTemplated.getVars(self)
+        vars["startTime"] = format_time(self._break.timetable_entry.start_dt)
+        vars["title"] = self._break.title
         return vars
 
 
-class WOverviewContribBase( wcomponents.WTemplated ):
+class WOverviewContribBase(wcomponents.WTemplated):
 
-    def __init__( self, aw, contrib, date, details="conference" ):
+    def __init__(self, aw, contrib, date, details="conference"):
         self._aw = aw
         self._contrib = contrib
         self._date = date
         self._details = details
 
-    def _getSpeakerText( self ):
-        l = []
-        if self._contrib.getSpeakerText()!="":
-            l.append( self._contrib.getSpeakerText() )
-        for av in self._contrib.getSpeakerList():
-            l.append( "%s"%av.getAbrName() )
-        speakers = ""
-        if len(l)>0:
-            speakers  = "(%s)"%"; ".join(l)
-        return speakers
-
-    def _getLocation( self ):
-        loc = ""
-        if self._contrib.getOwnLocation() != None:
-            loc = self._contrib.getLocation().getName()
-        room = ""
-        if self._contrib.getOwnRoom() != None:
-            if loc != "":
-                loc = "%s: "%loc
-            room = self._contrib.getRoom().getName()
-            url = linking.RoomLinker().getURL( self._contrib.getRoom(), \
-                                                   self._contrib.getLocation() )
-            if url != "":
-                room = """<a href="%s" style="font-size: 0.9em">%s</a>"""%(url, room)
-        if loc != "" or room != "":
-            return "(%s %s)"%(loc, room)
+    def _getSpeakerText(self):
+        speakers = [link.person.get_full_name(abbrev_first_name=True, last_name_upper=False)
+                    for link in self._contrib.person_links if link.is_speaker]
+        if speakers:
+            return u'({})'.format(u'; '.join(speakers))
         else:
-            return ""
+            return ''
 
-    def getHTML( self, params={} ):
-        return wcomponents.WTemplated.getHTML( self, params )
+    def getHTML(self, params={}):
+        return wcomponents.WTemplated.getHTML(self, params)
 
-    def getVars( self ):
-        vars = wcomponents.WTemplated.getVars( self )
-        tz = DisplayTZ(self._aw).getDisplayTZ()
-        vars["timezone"] = tz
-        vars["startTime"] = self._contrib.getAdjustedStartDate(tz).strftime( "%H:%M" )
-        vars["title"] = self._contrib.getTitle()
+    def getVars(self):
+        vars = wcomponents.WTemplated.getVars(self)
+        vars["timezone"] = DisplayTZ(self._aw).getDisplayTZ()
```

Comment:
@OmeGak  didn't you want to add another property to `Event` that returns the tzinfo as provided by DisplayTZ (i.e. the user's timezone if he doesn't want to use event timezones)